qiskit_dynamics/solvers/fixed_step_solvers.py

```diff
@@ -265,6 +268,7 @@ def jax_expm_solver(
     y0: Array,
     max_dt: float,
     t_eval: Optional[Union[Tuple, List, Array]] = None,
+    magnus_order: Optional[int] = 1,
```

Comment:
my understanding is that Optional[T] is equivalent to Union[T, None], so that Optional is actually incorrect here as it should always be an int.